pkg/skaffold/sync/sync.go

```diff
@@ -247,7 +248,11 @@ func Perform(ctx context.Context, image string, files map[string]string, cmdFn f
 
 				cmds := cmdFn(ctx, p, c, files)
 				for _, cmd := range cmds {
+					buf := &bytes.Buffer{}
+					cmd.Stdout = buf
+					cmd.Stderr = buf
 					if err := util.RunCmd(cmd); err != nil {
```

Comment:
@prary I wonder if `RunCmdOut` would do the job here?

Reply:
@dgageot , yes this thing also works, will update the PR.